File: mobile/src/main/java/treehou/se/habit/util/logging/Logger.java
package treehou.se.habit.util.logging;


public interface Logger {
    void d(String tag, String message);
    void i(String tag, String message);
    void w(String tag, String message);
    void w(String tag, String message, Throwable error);
    void e(String tag, String message);
    void e(String tag, String message, Throwable error);
}

File: mobile/src/main/java/treehou/se/habit/util/logging/Logger.kt
package treehou.se.habit.util.logging


interface Logger {
    fun d(tag: String, message: String)
    fun i(tag: String, message: String)
    fun w(tag: String, message: String)
    fun w(tag: String, message: String, error: Throwable)
    fun e(tag: String, message: String)
    fun e(tag: String, message: String, error: Throwable)
}
